### PR TITLE
remove a couple rules from loose

### PR DESCRIPTION
**Pull request by georgebonnr**
a couple pseudo-random changes.

Some of these I feel like I'd want in my IDE, but not in my console output...  maybe we can eventually make those configurable individually if there's enough of a need.

### CODE REVIEW

File: loose.json [hunk 1]
```diff
@@ -42,9 +40,8 @@
         "space-infix-ops": 1,
         "space-unary-ops": 1,
         "wrap-iife": [2, "inside"],
-        "react/display-name": 1,
```
Comment (georgebonnr) [inline loose.json:45]: jsx adds this automatically, according to dox

File: loose.json [hunk 2]
```diff
@@ -42,9 +40,8 @@
...
-        "react/jsx-boolean-value": 1,
-        "react/jsx-quotes": 1,
+        "react/jsx-boolean-value": [1, "never"],
```
Comment (georgebonnr) [inline loose.json:43]: we can remove this if it's annoying.

File: loose.json [hunk 3]
```diff
@@ -42,9 +40,8 @@
...
+        "react/jsx-quotes": [1, "double", "avoid-escape"],
```
Comment (georgebonnr) [inline loose.json:44]: this can also be removed if too annoying

**Comment by derekr** [comment]
LGTM

File: loose.json [hunk 4]
```diff
@@ -42,22 +40,17 @@
...
         "react/jsx-no-undef": 1,
-        "react/jsx-sort-props": 1,
-        "react/jsx-sort-prop-types": 1,
         "react/jsx-uses-react": 1,
         "react/jsx-uses-vars": 1,
         "react/no-did-mount-set-state": 1,
         "react/no-did-update-set-state": 1,
         "react/no-multi-comp": 1,
         "react/no-unknown-property": 1,
-        "react/prop-types": 1,
```
Comment (georgebonnr) [inline loose.json:57]: this seems to duplicate what react does in dev mode.
TODO: is there a reason to use one vs. the other?  Should we leave this on for prod builds?

File: strict.json [hunk 5]
```diff
@@ -45,11 +45,9 @@
         "space-unary-ops": 1,
         "wrap-iife": [2, "inside"],
         "react/display-name": 1,
-        "react/jsx-boolean-value": 1,
-        "react/jsx-quotes": 1,
+        "react/jsx-boolean-value": [1, "never"],
+        "react/jsx-quotes": [1, "double", "avoid-escape"],
         "react/jsx-no-undef": 1,
-        "react/jsx-sort-props": 1,
```
Comment (georgebonnr) [inline strict.json:51]: TODO: document / example some organic conventions for this

File: strict.json [hunk 6]
```diff
@@ -59,7 +57,6 @@
         "react/prop-types": 1,
         "react/react-in-jsx-scope": 1,
         "react/self-closing-comp": 1,
-        "react/sort-comp": 1,
```
Comment (georgebonnr) [inline strict.json:62]: again, use loose organic conventions for this?

File: loose.json [hunk 7]
```diff
@@ -42,22 +40,17 @@
...
         "react/react-in-jsx-scope": 1,
         "react/self-closing-comp": 1,
-        "react/sort-comp": 1,
-        "react/wrap-multilines": 1
+        "react/wrap-multilines": 2
```
Comment (georgebonnr) [inline loose.json:54]: As I recall the error msg in React when you forget to do this is cryptic.  So could be helpful to error on this explicitly?  We'll see

**Pull request merged**